README.rst

```diff
@@ -111,10 +111,25 @@ async/await fixtures
 pytest fixture semantics of setup, value, and teardown.  At present only
 function scope is supported.
 
+Note: in pytest-twisted version 0.12, you must *call*
+``pytest_twisted.async_fixture`` and ``pytest_twisted.async_yield_fixture``.
```

Comment:
```suggestion
``pytest_twisted.async_fixture()`` and ``pytest_twisted.async_yield_fixture()``.
```

Just to drive the point home?  And I suppose I should probably put the specified scope into the exception.  `scope <function foo> selected` might help out without even doing any type checking etc.

Reply:
Agreed on driving the point home.

The error already does that, kinda. From 0.12: `pytest_twisted.AsyncFixtureUnsupportedScopeError: Unsupported scope used for async fixture: <function foo at 0x108fccb00>`

But now I realize why that error confused me so much. After isolating the issue, I now read that as the thing after `:` being the improper scope, but beforehand I think I interpreted it as saying the fixture function `foo` had the wrong scope.